File: fire drill 1/javascript/TaskSeven.js

for(let count = 1; count <= 10; count++){
if(count % 4 == 0){
let sum = 0;
for(let counter = 1; counter <= 5; counter++){
	sum += (Math.pow(count, counter));
}
console.log(sum + " ");
}

}

File: fire drill 1/Java file/TaskSeven.java
public class TaskSeven{
public static void main(String[] args){

for(int count = 1; count <= 10; count++){
if(count % 4 == 0){
int sum = 0;
for(int counter = 1; counter <= 5; counter++){
	sum += ((int)Math.pow(count, counter));
}
System.out.print(sum + " ");
}

}

}

}
